.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Docker
+on:
+  create:
+    tags:
+      - .*
+  push:
+    branches:
+      - .*
```

Comment:
@Sammers21 Isn't is better to specify `master` branch only here?

Reply:
Pull request branches does not count, that is why i think it is ok

Reply:
@Sammers21 It actually should fine, the only edge case would be if someone creates extra branch directly.
Ok, let it be.